authcaptureproxy/auth_capture_proxy.py

```
@@ -382,24 +380,31 @@ async def _process_multipart(reader: MultipartReader, writer: MultipartWriter) -
             self.data.update(data)
             _LOGGER.debug("Storing data %s", data)
         elif json_data:
             self.data.update(json_data)
             _LOGGER.debug("Storing json %s", json_data)
         if URL(str(request.url)).path == re.sub(
-            r"/+", "/", self._proxy_url.with_path(f"{self._proxy_url.path}/stop").path
+            r"/ +", "/", self._proxy_url.with_path(f"{self._proxy_url.path}/stop").path
         ):
             self.all_handler_active = False
             if self.active:
                 asyncio.create_task(self.stop_proxy(3))
```

Ruff (0.14.14)

Store a reference to the return value of `asyncio.create_task`

(RUF006)